app/assets/stylesheets/osem-schedule.css.scss

```diff
@@ -75,3 +75,67 @@ a.unstyled-link {
   background-image: linear-gradient(to top, #f7faf2 24%, #53ab4a 97%,
 #53ab4a 100%);
 }
+
+.event-item{
```

Comment:
Please use bootstrap components and not your own styling. Makes it way easier to maintain in a group.